server/db/seedscript.js

```diff
@@ -50,8 +23,9 @@ db.on('error', console.error.bind(console, 'mongoose connection error:'));
 
 db.once('open', () => {
   console.log('successfully connected to mongoose');
-  (() => {
+  (async () => {
     let seeder = 1;
+
     while (seeder !== 101) {
```

Comment:
Not a necessary change at all but I'd change the condition from "seeder !== 101" to "seeder <= 100", its a little more self-explanatory of what your stop condition is since you want to populate 100 rows of data - either way this will work the same way